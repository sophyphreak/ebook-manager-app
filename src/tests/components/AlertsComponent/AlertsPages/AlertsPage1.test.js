import React from 'react';
import { shallow } from 'enzyme';
import AlertsPage1 from '../../../../components/AlertsComponent/AlertsPages/AlertsPage1';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AlertsPage1 />);
});

test('should render AlertsPage1 correctly', () => {
  expect(wrapper).toMatchSnapshot();
});