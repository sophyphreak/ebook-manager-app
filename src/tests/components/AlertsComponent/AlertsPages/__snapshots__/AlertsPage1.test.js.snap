// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AlertsPage1 correctly 1`] = `
<Card
  className="animated fadeIn"
  tag="div"
>
  <CardHeader
    tag="div"
  >
    <h4>
      <strong>
        Alerts - 1 of 3
      </strong>
    </h4>
  </CardHeader>
  <CardBody
    tag="div"
  >
    <Form
      tag="form"
    >
      <Title />
      <ASIN />
      <AmazonURL />
      <IsFiction />
      <Genre />
      <SubGenre />
      <FirstName />
      <LastName />
      <Email />
      <Button
        color="primary"
        size="sm"
        tag="button"
        type="submit"
        value="Submit"
      >
        Next Section
      </Button>
    </Form>
  </CardBody>
</Card>
`;
